docs/specification.md

```
@@ -169,66 +167,36 @@ PCブラウザでの利用を前提とし、まずは自分専用の最小構成
-
-### レベルテーブル
-
-| レベル | 必要累計EXP |
-|---|---:|
-| 1 | 0 |
-| 2 | 30 |
-| 3 | 70 |
-| 4 | 120 |
-| 5 | 200 |
-| 6 | 300 |
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)